### PR TITLE
chore(i18n): add Indonesian (id) language support

### PR DESCRIPTION
**Pull request by risqiromadhoni**
Added Indonesian translations for @valibot/i18n

### CODE REVIEW

**Comment by fabian-hiller** [comment]
Thank you for this PR! Do you know another native speaking developer that could review the translations for me?

**Comment by fabian-hiller** [comment]
Any updates on this PR?

**Comment by smujaddid** [comment]
> Thank you for this PR! Do you know another native speaking developer that could review the translations for me?

I'm also a native Indonesian speaker. I can help review this at the weekend if you need it.

**Comment by fabian-hiller** [comment]
That would be great! Thank you very much!

**Review by smujaddid — CHANGES_REQUESTED** [review]
I think these translations along with my suggestions should be enough for direct or word-to-word translations.
For alternative, we can use different words for these messages, which doesn't have to be word-to-word translations. For example:
```
integer: (issue) => `Bukan integer. Menerima ${issue.received}`
```

File: packages/i18n/src/id.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    decimal:        (issue) => `Desimal tidak valid: Menerima ${issue.received}`,
+    email:          (issue) => `Email tidak valid: Menerima ${issue.received}`,
+    emoji:          (issue) => `Emoji tidak valid: Menerima ${issue.received}`,
+    empty:          (issue) => `Panjang tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    endsWith:       (issue) => `Akhir tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    excludes:       (issue) => `Konten tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    finite:         (issue) => `Finite tidak valid: Menerima ${issue.received}`,
+    hash:           (issue) => `Hash tidak valid: Menerima ${issue.received}`,
+    hexColor:       (issue) => `Warna hex tidak valid: Menerima ${issue.received}`,
+    hexadecimal:    (issue) => `Hexadecimal tidak valid: Menerima ${issue.received}`,
```
[the remark this answers is not in the document]
Comment (risqiromadhoni) [reply packages/i18n/src/id.ts]: Updated @smujaddid

File: packages/i18n/src/id.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    maxLength:      (issue) => `Panjang maksimum tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    maxSize:        (issue) => `Ukuran maksimum tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    maxValue:       (issue) => `Nilai maksimum tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    mimeType:       (issue) => `Tipe MIME tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    minBytes:       (issue) => `Byte minimum tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    minLength:      (issue) => `Panjang minimum tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    minSize:        (issue) => `Ukuran minimum tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    minValue:       (issue) => `Nilai minimum tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    multipleOf:     (issue) => `Banyak tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
```
[the remark this answers is not in the document]
Comment (risqiromadhoni) [reply packages/i18n/src/id.ts]: Updated @smujaddid

File: packages/i18n/src/id.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    nonEmpty:       (issue) => `Panjang tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    notBytes:       (issue) => `Byte tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    notLength:      (issue) => `Panjang tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    notSize:        (issue) => `Ukuran tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    notValue:       (issue) => `Nilai tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    octal:          (issue) => `Octal tidak valid: Menerima ${issue.received}`,
```
[the remark this answers is not in the document]
Comment (risqiromadhoni) [reply packages/i18n/src/id.ts]: Updated @smujaddid

File: packages/i18n/src/id.ts [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    regex:          (issue) => `Format tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    safeInteger:    (issue) => `Safe integer tidak valid: Menerima ${issue.received}`,
+    size:           (issue) => `Ukuran tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    startsWith:     (issue) => `Awal tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
```
[the remark this answers is not in the document]
Comment (risqiromadhoni) [reply packages/i18n/src/id.ts]: Updated @smujaddid

File: packages/i18n/src/id.ts [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    bytes:          (issue) => `Byte tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    check:          (issue) => `Input tidak valid: Menerima ${issue.received}`,
+    checkAsync:     (issue) => `Input tidak valid: Menerima ${issue.received}`,
+    creditCard:     (issue) => `Kartu kredit tidak valid: Menerima ${issue.received}`,
+    cuid2:          (issue) => `Cuid2 tidak valid: Menerima ${issue.received}`,
+    decimal:        (issue) => `Desimal tidak valid: Menerima ${issue.received}`,
+    email:          (issue) => `Email tidak valid: Menerima ${issue.received}`,
+    emoji:          (issue) => `Emoji tidak valid: Menerima ${issue.received}`,
+    empty:          (issue) => `Panjang tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    endsWith:       (issue) => `Akhir tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
```
[the remark this answers is not in the document]
Comment (risqiromadhoni) [reply packages/i18n/src/id.ts]: Updated @smujaddid

File: packages/i18n/src/id.ts [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    hexColor:       (issue) => `Warna hex tidak valid: Menerima ${issue.received}`,
+    hexadecimal:    (issue) => `Hexadecimal tidak valid: Menerima ${issue.received}`,
+    imei:           (issue) => `IMEI tidak valid: Menerima ${issue.received}`,
+    includes:       (issue) => `Konten tidak valid: Diharapkan ${issue.expected} tetapi menerima ${issue.received}`,
+    integer:        (issue) => `Integer tidak valid: Menerima ${issue.received}`,
+    ip:             (issue) => `IP tidak valid: Menerima ${issue.received}`,
+    ipv4:           (issue) => `IPv4 tidak valid: Menerima ${issue.received}`,
+    ipv6:           (issue) => `IPv6 tidak valid: Menerima ${issue.received}`,
+    isoDate:        (issue) => `Tanggal tidak valid: Menerima ${issue.received}`,
+    isoDateTime:    (issue) => `Date-time tidak valid: Menerima ${issue.received}`,
```
[the remark this answers is not in the document]
Comment (risqiromadhoni) [reply packages/i18n/src/id.ts]: Updated @smujaddid

**Comment by fabian-hiller** [comment]
Does this PR look good to all of you now?

**Comment by smujaddid** [comment]
> Does this PR look good to all of you now?

Yes, looking good to me.

**Pull request merged**